3_javascript/javascript3/week2/script.js

```diff
@@ -0,0 +1,75 @@
+
+
+// Function to fetch data from the API
+async function fetchData() {
+    try {
+        const response = await fetch('https://open.er-api.com/v6/latest/USD');
+        const data = await response.json();
+        return data;
+    } catch (error) {
+        console.error('Error fetching data:', error);
+        return null;
+    }
+}
+
+// Function to populate dropdowns with currency options
+function populateDropdowns() {
+    const fromCurrencyDropdown = document.getElementById('fromCurrency');
+    const toCurrencyDropdown = document.getElementById('toCurrency');
+
+    // Clear existing options
+    fromCurrencyDropdown.innerHTML = '';
+    toCurrencyDropdown.innerHTML = '';
+
+    // Check if currencyList is defined
+    if (currencyList) {
+        // Populate 'From' and 'To' dropdowns with currency options
+        currencyList.forEach(currency => {
+            const option1 = document.createElement('option');
+            option1.value = currency;
+            option1.textContent = currency;
+            fromCurrencyDropdown.appendChild(option1);
+
+            const option2 = document.createElement('option');
```

Comment:
The same here, the variable name could be improved.

Reply:
Ok.